lib/connections/kcp_listen.go

```diff
@@ -285,11 +292,10 @@ func (t *kcpListener) stunRenewal(listener net.PacketConn) {
 					break
 				}
 			}
-
-			oldType = natType
 		}
 
-		// We failed to contact all provided stun servers, chillout for a while.
+		// We failed to contact all provided stun servers or the nat is not punchable.
+		// Chillout for a while.
 		time.Sleep(time.Minute)
```

Comment:
Constantify and raise? a minute is quite a short time isn't it? or maybe that's fine it's just opinion, but constantify at least.